runtime/arc.js

```diff
@@ -154,7 +154,7 @@ class Arc {
         case 'firebase':
```

Comment:
Why does this case not have `{}`?

Reply:
(I assume the rule is 1-line, and it counts 1-line+break as a still 1-line?)

Reply:
`{}` is required when a case statement needs a scope to define variables